spec/facets/ColumnLineageDatasetFacet.json

```diff
@@ -66,11 +66,11 @@
                   },
                   "transformationDescription": {
                     "type": "string",
-                    "description": "a string representation of the transformation applied"
+                    "description": "DEPRECATED; a string representation of the transformation applied"
```

Comment:
JSON Schema 2019 draft introduces `"deprecated": true` annotation:
https://json-schema.org/understanding-json-schema/reference/annotations

Can it be used here?

Reply:
looks good, I'll use it
thanks!